scripts/cc-cli/src/commands/withdrawUnbonded.ts

```diff
@@ -21,8 +16,8 @@ export function makeWithdrawUnbondedCommand() {
 async function withdrawUnbondedAction(options: OptionValues) {
   const { api } = await newApi(options.url);
 
-  const stashSeed = getSeedFromOptions(options);
-  const stashAccount = initKeyringPair(stashSeed);
+  const controllerSeed = await getControllerSeedFromEnvOrPrompt();
+  const stashAccount = initKeyringPair(controllerSeed);
```

Comment:
`stashAccount` -> `pair` (or keyringPair if you prefer).

Reply:
done